FIAT/nedelec_second_kind.py

```diff
@@ -133,19 +133,20 @@ def _generate_face_dofs(self, cell, degree, offset, variant, quad_deg):
 
         # Return empty info if not applicable
         d = cell.get_spatial_dimension()
-        if (degree < 2):
+        if degree < 2:
             return (dofs, ids)
 
         msg = "2nd kind Nedelec face dofs only available with UFC convention"
         assert isinstance(cell, UFCTetrahedron), msg
 
+        num_edge_pts = degree + 1 if interpolant_deg is None else interpolant_deg + 1
+
         # Iterate over the faces of the tet
         num_faces = len(cell.get_topology()[2])
         for face in range(num_faces):
 
             # Construct quadrature scheme for this face
-            m = 2 * (degree + 1)
-            Q_face = UFCTetrahedronFaceQuadratureRule(face, m)
+            Q_face = UFCTetrahedronFaceQuadratureRule(face, num_edge_pts)
```

Comment:
I'm still not liking the relation between the number of edge points and the interpolation degree here.  If we have facet quadrature rules, we should be specifying those by some kind of degree of accuracy?

Reply:
This one is a very specialized one, only used here, and I haven't made sure the degree here is the minumum for unisolvence.

Reply:
Should we be thinking about having the `UFCTetrahedronFaceQuadratureRule` hook into the `create_quadrature` then so we can specify degrees to it?